packages/react-native-worklets/src/shareables.ts

```diff
@@ -248,6 +255,38 @@ function cloneNull(): ShareableRef<null> {
   return WorkletsModule.makeShareableNull();
 }
 
+function cloneObjectProperties<T extends object>(
+  value: T,
+  shouldPersistRemote: boolean,
+  depth: number
+): Record<string, unknown> {
+  const clonedProps: Record<string, unknown> = {};
+  for (const [key, element] of Object.entries(value)) {
+    if (key === '__initData' && clonedProps.__initData !== undefined) {
+      continue;
```

Comment:
I know there was no such comment before but let's add one that explains why we continue here.